src/Http/Http/src/Internal/BufferingHelper.cs

```diff
@@ -49,5 +50,21 @@ public static HttpRequest EnableRewind(this HttpRequest request, int bufferThres
             }
             return section;
         }
+
+        public static Stream EnableRewind(this MultipartPipeSection section, Action<IDisposable> registerForDispose,
```

Comment:
Why public?

Reply:
The others are, but the whole type is internal so it doesn't matter.